scripts/src/test.sh

```diff
@@ -125,6 +125,26 @@ client_weights_test() {
   # Killing node process added in js script
 }
 
+# $1 - ROOT DIR
+uploads_test() {
+  ROOT_DIR="$1"
+  shift
+
+  TEST_SCRIPT_PATH="$ROOT_DIR/scripts/test-utils"
+
+  # Run node
+  RUST_LOG="gear=debug,gwasm=debug" $ROOT_DIR/target/release/gear-node \
+  --dev --tmp --unsafe-ws-external --unsafe-rpc-external --rpc-methods Unsafe --rpc-cors all & sleep 7
```

Comment:
`sleep 7` is old timer. We can use `sleep 3` now

Reply:
For me wait 4 seconds more to make sure, that any machine successfully ran node is ok, isn't it?

Reply:
Yep) just FYI

Reply:
Done